chain/client/src/stateless_validation/validate.rs

```diff
@@ -92,60 +101,64 @@ pub fn validate_chunk_endorsement(
     epoch_manager: &dyn EpochManagerAdapter,
     endorsement: &ChunkEndorsement,
     store: &Store,
-) -> Result<bool, Error> {
-    if !validate_chunk_relevant_as_validator(
+) -> Result<Result<(), ChunkNotRelevant>, Error> {
```

Comment:
I wonder if we can simplify `Result<(), ChunkNotRelevant>` by using an enum (e.g. ChunkRelevance, ChunkInclusionState, etc) that is the same as `ChunkNotRelevant` plus an "everything is fine" variant

Reply:
Might be a good idea.

Reply:
Done.